comp/core/health-platform/README.md

```diff
@@ -0,0 +1,180 @@
+# Health Platform Component
+
+The Health Platform component provides a mechanism for collecting and reporting health information from the Datadog Agent to the backend. It includes hostname information, host ID from gopsutil, organization ID, and a list of issues with their details.
+
+## Features
+
+- **Issue Management**: Add, remove, and list issues
+- **Periodic Reporting**: Automatically send health reports at configurable intervals
+- **On-Demand Reporting**: Manually trigger health reports
+- **Host Information**: Automatically includes hostname and host ID
+- **Organization Context**: Includes organization ID for proper routing
+- **Protobuf Serialization**: Uses efficient protobuf format for backend communication
+
+## Configuration
+
+The component supports the following configuration options:
+
+```yaml
+# Reporting interval (default: 15 minutes)
+health_platform:
+  interval: 15m
+
+# Organization ID (required)
+org_id: 12345
+# Alternative: API key org ID
+api_key_orgid: 12345
+```
+
+## Usage Example
+
+```go
+package main
+
+import (
+    "context"
+    "time"
+
+    "github.com/DataDog/datadog-agent/comp/core/health-platform"
+    "github.com/DataDog/datadog-agent/comp/core/health-platform/fx"
+)
+
+func example(healthPlatform healthplatform.Component) {
```

Comment:
I wonder if word platform is needed. Do we have many other components with word health?